### ISSUE TITLE
Deep Learning Paper Suggestions

### ISSUE THREAD

**Issue by sumodm** [sumodm/bcvm#1]
We will add the initial recommendations here and then move them to a papers.md as we start organizing the papers.

Application and Specific Architectures
1. [Hypercolumns for Object Segmentation and Fine-Grained Localization](http://arxiv.org/abs/1411.5752)
2. [Deep SimNets](http://arxiv.org/abs/1506.03059)

Theory
3. [On the Expressive Power of Deep Learning: A Tensor Analysis](http://arxiv.org/pdf/1509.05009v1.pdf)

Recurrent Papers
4. [LSTM: A Search Space Odyssey](http://arxiv.org/abs/1503.04069)

**Comment by prabhuteja12** [comment]
Just adding a list of papers that we can discuss and the finalized ones can be added to the homepage.

Week-3: Maybe [this](http://cs231n.github.io/convolutional-networks/) page for the basics. And [AlexNet](http://papers.nips.cc/paper/4824-imagenet-classification-with-deep-convolutional-neural-networks.pdf)

Week-4: [VGG Net](http://arxiv.org/pdf/1409.1556.pdf) and [Going Deeper with Convolutions](http://www.cs.unc.edu/~wliu/papers/GoogLeNet.pdf)

Week-5: This year's ILSVRC winner http://arxiv.org/pdf/1512.03385.pdf

**Comment by eyebies** [comment]
Here are a few papers that interest me. 
DeepFace: Closing the Gap to Human-Level Performance in Face Verification
Real-Time Pedestrian Detection With Deep Network Cascades
Region-based Convolutional Networks for Accurate Object Detection and Segmentation

Can we squeeze them in ?

**Comment by utkarshsimha** [comment]
[Batch Normalization](http://arxiv.org/abs/1502.03167)
[Deep Residual Networks](http://arxiv.org/abs/1512.03385)

**Comment by sumodm** [comment]
#### Saliency
- [NUS: Deep Learning for Saliency](https://www.ece.nus.edu.sg/stfpage/eleqiz/deep_saliency.html)
- [Saliency Detection by Multi-Context Deep Learning](http://www.cv-foundation.org/openaccess/content_cvpr_2015/papers/Zhao_Saliency_Detection_by_2015_CVPR_paper.pdf)
- [Large-Scale Optimization of Hierarchical Features for Saliency Prediction in Natural Images](http://coxlab.org/pdfs/cvpr2014_vig_saliency.pdf)

#### Visualization
- [Deep Inside Convolutional Networks: Visualizing Image Classification Models and Saliency Maps](http://arxiv.org/pdf/1312.6034.pdf)

#### Deeper Architectures
- [Deep Residual Learning for Image Recognition](http://arxiv.org/pdf/1512.03385v1.pdf)

#### Shallow Architectures
- [Highway Networks](http://arxiv.org/pdf/1505.00387v2.pdf)

#### Misc
- [Batch Normalization: Accelerating Deep Network Training by Reducing Internal Covariate Shift](http://arxiv.org/pdf/1502.03167.pdf)

#### Context
- [A Critical View of Context: Not Deep Learning](http://www.cs.tau.ac.il/~wolf/papers/context.pdf)

**Comment by sumitbinnani** [comment]
#### Limits of Computer Image Recognition

> _Atoms of Recognition in Human and Computer Vision_
> http://www.pnas.org/content/early/2016/02/09/1513198113.full.pdf
> 
> _Blog Post on the aforementioned paper_
> http://arstechnica.com/science/2016/02/tiny-blurry-pictures-find-the-limits-of-computer-image-recognition/

**Comment by utkarshsimha** [comment]
[He Initialization for Relu](http://arxiv.org/abs/1502.01852)

**Comment by IMG-PRCSNG** [comment]
[Fractal Nets - Alternative to Residual Nets](`https://arxiv.org/pdf/1605.07648v1.pdf`)
